PluginCore/PluginCore/FRService/FRSearch.cs

```diff
@@ -608,6 +609,45 @@ private List<SearchMatch> SearchSource(string src, int startIndex, int startLine
             return results;
         }
 
+        private bool LookupRegex(string ba, ref int i)
+        {
+            if (PluginBase.CurrentProject == null || 
```

Comment:
But regex literals are a special case, supported only on a handful of languages, but that need to be handled like comments and string literals, otherwise when FRSearch finds a quote inside a regex literal it thinks it's a string literal, and cause bugs like the one above.